### PR TITLE
Change appropriate "preview" references to "beta"

### PR DESCRIPTION
**Pull request by heaths**
Bookmarks and any reference to service preview versions were not changed.

### CODE REVIEW

**Comment by heaths** [comment]
I wasn't sure whether to change a script in there for Java, so I added a TODO. Should a doc or blurb be added as to why we decided to change?

File: docs/policies/releases.md [hunk 1]
```diff
@@ -142,30 +142,30 @@ Each language repo uses a badge for analysis of dependencies on `master` includi
...
-- `X.Y.ZbN` (preview release using beta convention)
+- `X.Y.ZbN` (beta release using beta convention)
 
-Preview packages will be published PyPi. Dev packages will be published to an isolated Azure DevOps feed.
+Beta packages will be published PyPi. Dev packages will be published to an isolated Azure DevOps feed.
 
 ##### Incrementing after release (Python)
 
-**Preview Release:**  `1.0.0b1` -> `1.0.0b2`
+**Beta Release:**  `1.0.0b1` -> `1.0.0b2`
```
Comment (weshaggard) [inline docs/policies/releases.md:168]: @heaths I'm torn about changing this language. It is still a preview release we just happen to use "beta" for the prerelease tag. Do you think it is better to call these beta releases? 

On a similar note I'm going to be changing the tags for "dev" builds to use "alpha" for the prerelease tag but I plan to still refer to them as daily dev releases as opposed to alpha releases.

Comment (heaths) [reply docs/policies/releases.md:168]: I see you're point, but worry that would be confusing (calling it "beta" some places and "preview" others), and could be cause for alarm as to why we are making this change in the first place. Seems better to scrub "preview" in as many places as possible. I just made sure to leave any references to service previews as-is (weren't many).

Comment (weshaggard) [reply docs/policies/releases.md:168]: I've updated my PR to refer to it as beta release anywhere we were talking about our libraries, and kept the work preview when referring to the service version.

File: eng/scripts/Update-Release-Versions.ps1 [hunk 2]
```diff
@@ -103,6 +103,7 @@ function Update-java-Packages($packageList)
       Write-Warning "Not updating VersionGA for $($pkg.Package) because at least one associated URL is not valid!"
     }
 
+    # TODO: Do references to "latest-preview" need to be changed to "latest-beta"?
```
Comment (weshaggard) [inline eng/scripts/Update-Release-Versions.ps1:106]: This one requires more work elsewhere before we can change. It is tied to the docs and they still refer to things as preview so I'm not likely going to change that.

**Comment by weshaggard** [comment]
Closing in favor of https://github.com/Azure/azure-sdk/pull/1536. I believe I have captured everything you have here in my PR but please let me know if I missed something.

**Pull request closed without merging**